pkg/blockstorage/awsebs/awsebs.go

```diff
@@ -736,6 +736,9 @@ func staticRegionToZones(region string) ([]string, error) {
 			"eu-central-1a",
 			"eu-central-1b",
 			"eu-central-1c",
+			"eu-central-1-wl1-ber-wlz-1", 
```

Comment:
That's an interesting naming scheme.
Is there an official documentation link for these?

Reply:
The names are defined by AWS. We call the list regions API and these are the values it provides.

Reply:
This is what the AWS CLI spews out:

```
aws ec2 describe-availability-zones --region eu-central-1 |jq .AvailabilityZones[].ZoneName

"eu-central-1a"
"eu-central-1b"
"eu-central-1c"

```

Reply:
@julio-lopez The result varies by the credentials I think. Another reason why this should be dynamic.

Reply:
x-reference: 

```
aws ec2 describe-availability-zones --all-availability-zones --region eu-central-1 |jq .AvailabilityZones[].ZoneName
"eu-central-1a"
"eu-central-1b"
"eu-central-1c"
"eu-central-1-wl1-ber-wlz-1"
"eu-central-1-wl1-dtm-wlz-1"
"eu-central-1-wl1-muc-wlz-1"
```